src/backend/codegen/CMakeLists.txt

```diff
@@ -113,7 +127,7 @@ include_directories(${CLANG_INCLUDE_DIRS})
 # Core codegen library.
 add_library(gpcodegen SHARED
             utils/clang_compiler.cc
-            utils/code_generator.cc
+            utils/codegen_utils.cc
```

Comment:
These may be codegen utils. But more specifically - this class is a wrapper around llvm - it's IRBuilder, IROptimizer etc. I bring this up because both clang_compiler and code_generator are codegenutils which is why they have that namespace. This name should be more instructive in my opinion.